api-response.go

```diff
@@ -17,16 +17,198 @@
 package main
 
 import (
+	"encoding/xml"
 	"net/http"
 
 	"github.com/minio/minio/pkg/fs"
 )
 
-// Reply date format
 const (
+	// Reply date format
 	rfcFormat = "2006-01-02T15:04:05.000Z"
+	// Limit number of objects in a given response.
+	maxObjectList = 1000
 )
 
+// LocationResponse - format for location response.
+type LocationResponse struct {
+	XMLName  xml.Name `xml:"http://s3.amazonaws.com/doc/2006-03-01/ LocationConstraint" json:"-"`
+	Location string   `xml:",chardata"`
+}
+
+// AccessControlPolicyResponse - format for get bucket acl response.
+type AccessControlPolicyResponse struct {
+	XMLName xml.Name `xml:"http://s3.amazonaws.com/doc/2006-03-01/ AccessControlPolicy" json:"-"`
+
+	AccessControlList struct {
+		Grant []Grant
```

Comment:
Shouldn't array of Grant be called Grants? Are you keeping it this way for XML generation?